src/igmpproxy.c

```diff
@@ -104,40 +102,39 @@ int main( int ArgCn, char *ArgVc[] ) {
 
     // Chech that we are root
     if (geteuid() != 0) {
-       fprintf(stderr, "igmpproxy: must be root\n");
-       exit(1);
+        fprintf(stderr, "igmpproxy: must be root\n");
+        exit(1);
     }
 
     openlog("igmpproxy", LOG_PID, LOG_USER);
 
     // Write debug notice with file path...
-    my_log(LOG_DEBUG, 0, "Searching for config file at '%s'" , configFilePath);
+    my_log(LOG_DEBUG, 0, "Searching for config file at '%s'", configFilePath);
 
     do {
 
         // Loads the config file...
-        if( ! loadConfig( configFilePath ) ) {
+        if (!loadConfig(configFilePath)) {
             my_log(LOG_ERR, 0, "Unable to load config file...");
             break;
         }
 
         // Initializes the deamon.
-        if ( !igmpProxyInit() ) {
+        if (!igmpProxyInit()) {
             my_log(LOG_ERR, 0, "Unable to initialize IGMPproxy.");
             break;
         }
 
-        if ( !Log2Stderr ) {
+        if (!Log2Stderr) {
 
             // Only daemon goes past this line...
-            if (fork()) exit(0);
+            if (fork())
+                exit(0);
 
             // Detach daemon from terminal
-            if ( close( 0 ) < 0 || close( 1 ) < 0 || close( 2 ) < 0
-                || open( "/dev/null", 0 ) != 0 || dup2( 0, 1 ) < 0 || dup2( 0, 2 ) < 0
-                || setpgid( 0, 0 ) < 0
-            ) {
-                my_log( LOG_ERR, errno, "failed to detach daemon" );
+            if (close(0) < 0 || close(1) < 0 || close(2) < 0 || open("/dev/null", 0) != 0 || dup2(0, 1) < 0 ||
+                dup2(0, 2) < 0 || setpgid(0, 0) < 0) {
```

Comment:
previous wrapping to 3 lines was better. close() calls on one line, open+dup() on second line and setpgid() on third.